android/beagle/src/main/java/br/com/zup/beagle/android/components/Image.kt

```diff
@@ -88,27 +90,29 @@ data class Image(
 
     private fun loadRemoteImage(imageView: ImageView, pathType: ImagePath.Remote) {
         val placeholder = pathType.placeholder?.mobileId
-        val requestOptions = getGlideRequestOptions(placeholder)
-        imageView.loadImage(pathType, requestOptions)
+        setPlaceHolder(placeholder, imageView)
+        imageView.loadImage(pathType)
     }
 
-    @SuppressLint("CheckResult")
-    private fun getGlideRequestOptions(placeholder: String?): RequestOptions {
-        val requestOptions = RequestOptions()
+    private fun setPlaceHolder(placeholder: String?, imageView: ImageView) {
         getImage(placeholder)?.let {
-            requestOptions.placeholder(it)
+            imageView.setImageDrawable(ContextCompat.getDrawable(imageView.context, it))
```

Comment:
What do you think of using another method so you don't need to do this type conversion?
```suggestion
            imageView.setImageResource(it)
```